plugins/footnote/config/settings.yml

```diff
@@ -0,0 +1,7 @@
+plugins:
+  enable_markdown_footnotes:
+    default: true
+    client: true
+  display_footnotes_inline:
+    default: true
+    client: true
```

Comment:
I wonder if this is a problem... it'll be default on for all instances now (whereas before you'd have to install the plugin). Let's discuss internally.